apps/meteor/server/models/raw/Rooms.ts

```diff
@@ -294,7 +294,10 @@ export class RoomsRaw extends BaseRaw<IRoom> implements IRoomsModel {
 				},
 			],
 			prid: { $exists: false },
-			$and: [{ $or: [{ federated: { $exists: false } }, { federated: false }] }],
+			$and: [
+				{ $or: [{ federated: { $exists: false } }, { federated: false }] },
+				{ $or: [{ archived: { $exists: false } }, { archived: false }] },
+			],
```

Comment:
```suggestion
			$and: [{ federated: { $ne: true } }, { archived: { $ne: true } }],
```
Your suggestion works, but I think it's more concise and readable like this.